.github/workflows/cuda.yml

```diff
@@ -60,6 +60,25 @@ jobs:
         PYWARPX_LIB_DIR=$PWD/build_sp/lib python3 -m pip wheel .
         python3 -m pip install *.whl
 
+  build_nvcc_gnumake:
+    name: NVCC 11.0.2 GNUmake
+    runs-on: ubuntu-18.04
+    if: github.event.pull_request.draft == false
+    steps:
+    - uses: actions/checkout@v2
+    - name: install dependencies
+      run: |
+        .github/workflows/dependencies/nvcc11.sh
+    - name: build WarpX
+      run: |
+        export PATH=/usr/local/nvidia/bin:/usr/local/cuda/bin:${PATH}
+        export LD_LIBRARY_PATH=/usr/local/nvidia/lib:/usr/local/nvidia/lib64:/usr/local/cuda/lib64:${LD_LIBRARY_PATH}
+        which nvcc || echo "nvcc not in PATH!"
+
+        git clone https://github.com/ECP-WarpX/picsar.git ../
+        git clone https://github.com/AMReX-Codes/amrex.git ../
```

Comment:
Uff, we need to add this to our update tools, otherwise we will get uncontrolled updates in CI, that block PRs:
https://github.com/ECP-WarpX/WarpX/tree/development/Tools/Release